packages/react/src/components/CalendarCell/CalendarCell.styles.ts

```diff
@@ -207,5 +202,22 @@ export const useStyles = css({
         },
       },
     },
+    {
```

Comment:
@branclar I'm struggling to get this compound variant actually to apply to the element -- let me know if anything jumps out at you!